index.js

```diff
@@ -273,6 +273,26 @@ class ScmBase {
     }
 
     /**
+    * Get list of opened PRs of a pipeline
```

Comment:
we just want the name? Probably also the`prRef` to get configurations for each PR